common/src/test/java/com/revenuecat/purchases/common/BaseHTTPClientTest.kt

```diff
@@ -88,10 +90,18 @@ abstract class BaseHTTPClientTest {
                 eTagHeader = any(),
                 "/v1${endpoint.getPath()}",
                 refreshETag = false,
+                requestDate = requestDateHeader,
                 verificationResult = verificationResult
             )
         } returns expectedResult
-        val response = MockResponse().setBody(expectedResult.payload).setResponseCode(expectedResult.responseCode)
+        val response = MockResponse()
+            .setBody(expectedResult.payload)
+            .setResponseCode(expectedResult.responseCode)
+            .apply {
+                if (requestDateHeader != null) {
+                    setHeader(HTTPResult.REQUEST_TIME_HEADER_NAME, requestDateHeader.time)
```

Comment:
Is `.time` ms?

Reply:
Yup, that's the official way to get milliseconds from a `Date`... I guess I could add a `Date.epochMilliseconds` extension or something like that to make it clearer, but it seems weird considering that's what this official API does.

Reply:
No it's fine, just making sure!